.gitignore

```
@@ -21,6 +21,11 @@
 
 # exclude data from source control by default
 /data/
+cts.csv
+mris.csv
+patients.csv
+xrays.csv
+
```

I think here you can just say 
```suggestion
*.csv
```
Doesn't feel like there's need to be more specific.